### DISCUSSION TITLE
Problem installing alpha 23

### DISCUSSION THREAD

**Discussion by Shosha555** [linuxdeepin/developer-center#3493] [General | 常规讨论]
![Screenshot_2022-11-16-10-30-58-214_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202129984-c0755010-8b45-4d3e-a109-f4e9b9d70934.jpg)
![Screenshot_2022-11-16-10-31-24-345_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202130005-089fa62d-f26c-4c21-973c-c83baa9a7204.jpg)
![Screenshot_2022-11-16-10-31-47-150_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202130018-7604c746-b386-431c-be75-a69ef479898c.jpg)
![Screenshot_2022-11-16-10-33-03-325_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202130032-e7ce0fc7-f135-4519-b0d8-4eb70c128181.jpg)
![Screenshot_2022-11-16-10-33-33-982_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202130047-05f0eb21-1f30-441a-93d8-614e94e79bc0.jpg)
![Screenshot_2022-11-16-10-34-05-163_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202130056-209feda6-8f1b-465c-9363-ee7efc5f1a3d.jpg)
![Screenshot_2022-11-16-10-34-29-789_com miui videoplayer](https://user-images.githubusercontent.com/118334949/202130071-5747bccb-ebda-49b4-b058-c155b128f9e6.jpg)
Hello, my friends, I downloaded Alpha 23, but I had a problem with the system installer. It was no longer the same as the 20.7 version. I chose Advanced, and then chose the partition to install the system on, and then chose ext4 And in the next box, choose / and press OK, and the system was installed smoothly and easily. In Alpha, the situation is different, and I take the same steps to no avail. Please fix the problem by updating the installer, or go back to install the previous system

**Comment by BLumia** [comment]
Is there any error message about the issue you've got? Photo or video would be helpful if you are not sure about where to look at.

**Reply by Shosha555** [reply]
My friend, I have attached pictures of the problem, thank you

**Reply by BLumia** [reply]
Thanks for the updated info! cc @babyfengfjx

**Comment by babyfengfjx** [comment]
Your bios is in EFI mode. If you use the advanced installation mode, you must divide an EFI partition first. You can divide an additional EFI partition to complete the normal installation：
![录屏_选择区域_20221117095138](https://user-images.githubusercontent.com/24642999/202334574-24647971-ea77-44f9-b7c4-6ed650105ef0.gif)

**Reply by Shosha555** [reply]
![IMG_20221117_232200](https://user-images.githubusercontent.com/118334949/202568488-f2332d80-21c2-4fd9-8d0b-34b16ce87cd2.jpg)
![IMG_20221117_232220](https://user-images.githubusercontent.com/118334949/202568551-2a63c74d-02ff-4b10-9bf6-4473a7780512.jpg)
![IMG_20221117_232237](https://user-images.githubusercontent.com/118334949/202568695-e8999d74-6018-42a5-9be3-17bfadaac06d.jpg)
![IMG_20221117_232309](https://user-images.githubusercontent.com/118334949/202568865-8d5fb416-0432-490b-a1d1-311e8873f69c.jpg)
There is no ( /boot/ efi ) in the installer options
There is no ( vfat ) in the installer options
info! cc @babyfengfjx

**Reply by babyfengfjx** [reply]
![image](https://user-images.githubusercontent.com/24642999/202938988-be4f26bd-43c0-43de-8324-04bcb6997739.png)
****
I probably understand that your BIOS is set to legacy mode, not EFI mode, so there will be no /boot/efi partition, at this time you can create a /boot partition, of course, this partition needs to be created in the first disk partition
